.storybook/cmsTheme.ts

```diff
@@ -3,33 +3,11 @@ import { create } from '@storybook/theming';
 export default create({
   base: 'light',
 
-  colorPrimary: '#112e51',
   colorSecondary: '#0071bc',
```

Comment:
What would happen if you make this string `var(--color-primary)` or something?

Reply:
Storybook errors: 

> Uncaught Error: Couldn't parse the color string. Please provide the color as a string in hex, rgb, rgba, hsl or hsla notation.

Reply:
Ah okay